models/magiceden/solana/magiceden_solana_events.sql

```diff
@@ -95,7 +95,11 @@ SELECT
 FROM {{ source('solana','transactions') }}
 LEFT JOIN prices.usd p
   ON p.minute = date_trunc('minute', block_time)
+  AND p.blockchain is NULL
```

Comment:
If we don't have this we can get multiple rows per line if another `SOL` is added in the future.

Reply:
Thanks ! I'll review this and run it on my end as well tmw 👍